x-pack/plugins/aiops/public/application/utils/query_utils.test.ts

```diff
@@ -0,0 +1,254 @@
+        {
+          range: {
+            '@timestamp': {
+              format: 'epoch_millis',
+              gte: 1640082000012,
+              lte: 1640103600906,
+            },
+          },
+        },
+        { match_all: {} },
+      ]);
+    });
+
+    it('includes a term filter when including a selectedChangePoint', () => {
+      const baseFilterCriteria = buildBaseFilterCriteria(
+        '@timestamp',
+        1640082000012,
+        1640103600906,
+        { match_all: {} },
+        selectedChangePointMock
+      );
+
+      expect(baseFilterCriteria).toEqual([
+        {
+          range: {
+            '@timestamp': {
+              format: 'epoch_millis',
+              gte: 1640082000012,
+              lte: 1640103600906,
+            },
+          },
+        },
+        { match_all: {} },
+        { term: { 'meta.cloud.instance_id.keyword': '1234' } },
+      ]);
+    });
+
+    it('includes a term filter with must_not when excluding a selectedChangePoint', () => {
+      const baseFilterCriteria = buildBaseFilterCriteria(
+        '@timestamp',
+        1640082000012,
+        1640103600906,
+        { match_all: {} },
+        selectedChangePointMock,
+        false
+      );
+
+      expect(baseFilterCriteria).toEqual([
+        {
+          range: {
+            '@timestamp': {
+              format: 'epoch_millis',
+              gte: 1640082000012,
+              lte: 1640103600906,
+            },
+          },
+        },
+        { match_all: {} },
+        { bool: { must_not: [{ term: { 'meta.cloud.instance_id.keyword': '1234' } }] } },
+      ]);
+    });
+
+    it('includes multiple term filters when including a selectedGroupMock', () => {
+      const baseFilterCriteria = buildBaseFilterCriteria(
+        '@timestamp',
+        1640082000012,
+        1640103600906,
+        { match_all: {} },
+        undefined,
+        true,
+        selectedGroupMock
+      );
+
+      expect(baseFilterCriteria).toEqual([
+        {
+          range: {
+            '@timestamp': {
+              format: 'epoch_millis',
+              gte: 1640082000012,
+              lte: 1640103600906,
+            },
+          },
+        },
+        { match_all: {} },
+        {
+          term: {
+            'error.message': 'rate limit exceeded',
+          },
+        },
+        {
+          term: {
+            message: 'too many requests',
+          },
+        },
+        {
+          term: {
+            'user_agent.original.keyword': 'Mozilla/5.0',
+          },
+        },
+        {
+          term: {
+            'beat.hostname.keyword': 'ip-192-168-1-1',
+          },
+        },
+        {
+          term: {
+            'beat.name.keyword': 'i-1234',
+          },
+        },
+        {
+          term: {
+            'docker.container.id.keyword': 'asdf',
+          },
+        },
+      ]);
+    });
+
+    it('includes a must_note with nested term filters when excluding a selectedGroupMock', () => {
```

Comment:
nit: typo here for `must_not`

Reply:
Fixed in [5f8e186](https://github.com/elastic/kibana/pull/141213/commits/5f8e186dab30d5e33613428143a4034b1214154d).